packages/block-library/src/calendar/style.scss

```
@@ -1,5 +1,7 @@
 .wp-block-calendar {
 	text-align: center;
+	// This block has customizable border-box makes that more predictable.
```

```suggestion
	// This block has customizable borders. Border-box sizing makes that more predictable.
```
There's a typo in this comment. Perhaps it should read something like above?